plugins/database/postgresql/postgresql_test.go

```diff
@@ -156,6 +156,12 @@ func TestPostgreSQL_CreateUser(t *testing.T) {
 				GRANT SELECT ON ALL SEQUENCES IN SCHEMA public TO "{{username}}";`,
 			},
 		},
+		"reproduce https://github.com/hashicorp/vault/issues/6098": {
+			createStmts: []string{
+				// NOTE: "rolname" in the following line is not a typo.
+				"DO $$ BEGIN IF NOT EXISTS (SELECT FROM pg_catalog.pg_roles WHERE rolname='my_role') THEN CREATE ROLE my_role; END IF; END $$",
```

Comment:
Could you template this entry so we can make sure everything around replacing usenames/passwords work ok?

Reply:
This is added in https://github.com/hashicorp/vault/pull/8512/commits/5a3dbbb87c2508fcd48f0aac426894d0331f128a

Reply:
Also, there are a bunch of existing tests in this file that cover templated fields.